examples/kafka-streams-examples/README.md

```diff
@@ -0,0 +1,230 @@
+# Confluent's Kafka Streams Examples
```

Comment:
My latest commits add a tutorial that will go under docs.confluent.io (alongside the other docker tutorials).  Also, I updated the README for this PR's example to point to docs.confluent.io.

TODO for @samjhecht et al: Some version identifiers are hardcoded in e.g. the README, e.g. `http://docs.confluent.io/3.2.0/and/so/on`.  Updating those version identifiers should be done as part of the release process.